### PR TITLE
Video encoding improvements

### PR DESCRIPTION
**Pull request by eddyStreamlabs**

### CODE REVIEW

File: app/components/windows/EditStreamInfo.vue [hunk 1]
```diff
@@ -30,11 +30,20 @@
...
             <div class="checkbox">
-              <input
-               type="checkbox"
-               v-model="useOptimizedProfile"
-              />
-              <label><span>Use optimized encoder settings for this game</span></label>
+              <div v-if="!genericProfiles">
+                <input
+                type="checkbox"
+                v-model="useOptimizedProfile"
```
Comment (andycreeth) [inline app/components/windows/EditStreamInfo.vue]: You're going to want to indent `type` and `v-model` an additional level.  Apparently prettier auto formatting for vue templates is coming soon.

File: app/components/windows/EditStreamInfo.vue [hunk 2]
```diff
@@ -30,11 +30,20 @@
...
+                />
+                <label><span>Use optimized encoder settings for this game</span></label>
+              </div>
+              <div v-else>
+                <input
+                type="checkbox"
+                v-model="useOptimizedProfile"
```
Comment (andycreeth) [inline app/components/windows/EditStreamInfo.vue]: Same issue with indents.

File: app/services/video-encoding-optimizations/definitions.ts [hunk 3]
```diff
@@ -32,15 +33,15 @@ export interface IProfile {
 
 const CPU_profile: IProfile = {
   profile: 'CPU',
-  description: 'CPU optimized',
-  longDescription: 'Reduction of the CPU load for an average video quality',
+  description: 'Medium',
+  longDescription: 'Optimized profile for average CPU',
```
Comment (andycreeth) [inline app/services/video-encoding-optimizations/definitions.ts]: "Optimized profile for average CPUs"

File: app/services/video-encoding-optimizations/definitions.ts [hunk 4]
```diff
@@ -32,15 +33,15 @@ export interface IProfile {
...
   preset: PresetType.veryfast
 };
 
 const VQ_profile: IProfile = {
   profile: 'VQ',
-  description: 'Video quality optimized',
-  longDescription: 'Improvement of the video quality at a very low CPU usage',
+  description: 'Low',
+  longDescription: 'Optimized profile for weak CPU',
```
Comment (andycreeth) [inline app/services/video-encoding-optimizations/definitions.ts]: "Optimized profile for weak CPUs"

File: app/components/windows/EditStreamInfo.vue (hunk 2 above)
Comment (eddyStreamlabs) [reply app/components/windows/EditStreamInfo.vue]: Done.

File: app/services/video-encoding-optimizations/definitions.ts (hunk 3 above)
Comment (eddyStreamlabs) [reply app/services/video-encoding-optimizations/definitions.ts]: Done.

File: app/services/video-encoding-optimizations/definitions.ts (hunk 4 above)
Comment (eddyStreamlabs) [reply app/services/video-encoding-optimizations/definitions.ts]: Done.

**Pull request merged**